Changelog improvements before the 3.2 release (#5905)
Mbed-TLS/mbedtls: ChangeLog.d/mbedtls_tlsver_enum.txt

New deprecations
   * Deprecate mbedtls_ssl_conf_max_version() and
     mbedtls_ssl_conf_min_version() in favor of
     mbedtls_ssl_conf_max_tls_version() and
     mbedtls_ssl_conf_min_tls_version().

Features

>    * The new functions mbedtls_ssl_conf_max_tls_version() and
>      mbedtls_ssl_conf_min_tls_version() use a single value to specify
>      the protocol version.

Reviewer: Presumably the section contents are coalesced for the `ChangeLog`, so these two entries won't necessarily be adjacent. In which case it would be better to use the standard "Add function foo()"-style:

```suggestion
   * Add functions mbedtls_ssl_conf_max_tls_version() and
     mbedtls_ssl_conf_min_tls_version() that use single values to specify
     protocol versions.
```

Author: Why is “Add functions …” preferable to “The new functions …”? We've been using both.

Reviewer 2: I would say that "The new functions" does not imply that there was a change that added them. It might mean that these new functions **now** `use a single value to specify [...]`. I agree with Tom here.

Author: I don't get this. “New” means it wasn't there before the change. It's a common way to word changelog entries (example: [Emacs](https://www.gnu.org/savannah-checkouts/gnu/emacs/news/NEWS.27.1) — mostly under “Lisp changes”).

Reviewer 2: I understand your approach, but even in the given example a lot of changes follow one of these rules:
- `The new function xxx has been added`, `The new xxx replaces the current [...]`
- headline: `New function xxx`, then, in the body `which is`, or just some description.

I would say that in fact the form proposed is also used, so I'm more inclined to accept it, however I still like other versions more. 
Given that I'm not a native speaker, I'll just let others speak in this specific change request.
